code/addons/storyshots-core/stories/storyshot.enzyme.test.js

```diff
@@ -1,7 +1,11 @@
 import path from 'path';
-import { mount } from 'enzyme';
+import { mount, configure } from 'enzyme';
+// @ts-expect-error (Converted from ts-ignore)
```

Comment:
I know this is copy/paste from the init script, but I think we should install `@types/enzyme-adapter-react-16` instead of ignoring the error.

Reply:
I care extremely little about it, it's storyshots + an internal test + a test in combination with enzyme.

I don't want to spend more time on enzyme, I was considering remove this test as well, I'm not sure it's adding much to our true coverage...